playwright/pages/components/BasePage.ts

```diff
@@ -0,0 +1,39 @@
+import { Locator, Page } from "@playwright/test";
+
+import { Menu } from "./Menu";
+
+export class BasePage {
+  readonly page: Page;
```

Comment:
It's POM model, everything in POM model is based on a "page". So Page is from playwright, but we want to share some stuff for all of our pages. So that is the BasePage. It could be renamed if it's not clear. Like `SharedPage`, `SliceMachinePage`, ect.. WDYT?

Reply:
- Page (playwright)
- SliceMachinePage (us)
- SlicesListPage (us)

Reply:
Done